HARMONIZED_TEST_ENVIRONMENT.md

```diff
@@ -0,0 +1,72 @@
+Harmonized Xroad test environment
+===================
+
+- LXD based common test development environment to Finland and Estonia
+	- Includes servers: CA, CENTRAL SERVER, MANAGEMENT SECURITY SERVER, SECURITY SERVERS
+	    - https://github.com/ria-ee/X-Road/tree/develop/ansible
+- Can be used for testing:
+	- UI and integrations, SOAP testing, Service testing and Service mocks
+	- Not for performance testing!
+- Benefits:
+	- Tests are compatible in Estonia and Finland common test environments
```

Comment:
Grammatical fix: Tests are compatible between Estonia and Finland common test environments

Reply:
changed